dashboard/.env

```diff
@@ -0,0 +1,7 @@
+REACT_APP_DATASETLIST=http://10.1.170.201/api/v1/datasets/list
```

Comment:
Although it's great to get something in, and we can refine it later, I'd prefer that you move quickly towards using `/api/v1/endpoints` rather than hardcoding access to my test server. Even if you're not ready to figure out how to leverage that off the initial page load (e.g., essentially `window.endpoints = fetch('/api/v1/endpoints')` in `index.html` as I'd had in the old dashboard), at least something more like `REACT_APP_SERVER=http://10.1.170.201` with (perhaps in data store initialization or at the `App.js` top level?), even if it's initially something simple like `endpoints = {REACT_APP_SERVER}/api/v1/endpoints`. Then the components would be accessing `endpoints.api.register`, `endpoints.api.login`, and so forth.